.circleci/config.yml

```diff
@@ -89,3 +94,41 @@ jobs:
               npm version prerelease --preid="${CIRCLE_BUILD_NUM}.${CIRCLE_BRANCH}.`git rev-parse --short HEAD`" --no-git-tag-version
               npm publish --tag current
             fi
+
+  visual:
+    docker:
+      - image: circleci/node:8
+    environment:
+      TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+    steps:
+      - run:
```

Comment:
Instead of copy pasting these steps, I would prefer to extract them to commands:
https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands